app/models/host_config_group.rb

```diff
@@ -1,7 +1,6 @@
 class HostConfigGroup < ApplicationRecord
   include Authorizable
   audited :associated_with => :host
-  audited :associated_with => :hostgroup
```

Comment:
note for myself: this was not doing anything before anyways, since audited was already loaded.

Reply:
:dart: